app/change_set_persisters/change_set_persister/characterize.rb

```diff
@@ -17,5 +16,9 @@ def run
         ::CharacterizationJob.set(queue: change_set_persister.queue).perform_later(file_set.id.to_s)
       end
     end
+
+    def created_file_sets
+      change_set.respond_to?(:created_file_sets) ? change_set.created_file_sets : []
```

Comment:
would `change_set.try(:created_file_sets) || []` also work? I know you hate ternaries...

Reply:
That's a hidden ternary! It's ternaries all the way down!

But yeah, sure.